tests/LineBot.Tests/LineBotTests/TheGetEventsMethod.cs

```diff
@@ -41,6 +41,21 @@ public async Task ShouldLogRequest()
 
                 Assert.AreEqual(expected, actual);
             }
+
+            [TestMethod]
+            public async void ShouldHaveDestination()
+            {
+                var logger = new TestLogger();
+                var bot = TestConfiguration.CreateBot(logger);
+                var request = new TestHttpRequest(JsonDocuments.Events.Webhook);
+
+                await bot.GetEvents(request);
+
+                var actual = Encoding.UTF8.GetString(logger.LogReceivedEventsEventsData);
+                Assert.Fail();
+
+                // TODO: Implement this test.
```

Comment:
You need to check if `events` from `events = await bot.GetEvents(request);` has the correct description.

Reply:
![image](https://user-images.githubusercontent.com/33629087/48260319-1a630580-e46f-11e8-833d-462418b9ddaa.png)
> this line is making the tests fail.

```
Message: Test method Line.Tests.LineBotTests+TheGetEventsMethod.ShouldHaveDestination threw exception: 
Newtonsoft.Json.JsonSerializationException: Could not create an instance of type Line.ILineEvent.
Type is an interface or abstract class and cannot be instantiated. Path 'events[0].replyToken', 
line 5, position 19.
```
> and this is the output

Reply:
Will be resolved when you change the Events property to `List<LineEvent>` instead. Missed that in an earlier review.

Reply:
Well If you see the latest commits, you'll see if `List<LineEvent>` is used then LineEvent must be made public and doing so will include a lot of documentation along with XML comments. So if you have any other solution to this problem, I can think about that.
![image](https://user-images.githubusercontent.com/33629087/48270060-2c53a100-e48d-11e8-94a4-b7321a03297b.png)